### DISCUSSION TITLE
番剧页更新后黑夜模式出现bug

### DISCUSSION THREAD

**Discussion by leheheQAQ** [the1812/Bilibili-Evolved#4186] [功能建议 / Ideas]
您好B站更新了番剧页的布局
导致黑暗模式在番剧页失效了
![image](https://user-images.githubusercontent.com/109117717/236675295-d720fc53-4bb3-45c9-a6bd-9a5d5059fc7c.png)

**Comment by Geeyun-JY3** [comment]
新版番剧页没适配。

bug 去开 issue。

**Comment by the1812** [comment]
~~居然又更新了, 上上个版本刚适配完一次 (这个我还没灰度到)~~

现在有了, 真就刷 KPI 呗

**Reply by Geeyun-JY3** [reply]
其实我被灰度到这个大改的新版番剧播放页面也有至少一个星期了，懒且感觉阿 B 还会改就没报（
